openadapt/utils.py

```diff
@@ -20,6 +20,22 @@
 
 EMPTY = (None, [], {}, "")
 
+def get_now_dt_str(dt_format=config.DT_FMT):
```

Comment:
Please add two newlines between before and after functions outside of classes (I believe `black` does this automatically`)

Reply:
resolved in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/9dc1850918e81648b9885dd8a1be836d5e8d95ef